easybuild/easyconfigs/a/aNCI/aNCI-2.0-iccifort-2019.5.281.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
+
+name = 'aNCI'
+version = '2.0'
+
+homepage = 'https://www.lct.jussieu.fr/pagesperso/contrera/nci-MD.html'
+description = """Non-covalent interaction (NCI) for MD trajectories"""
+
+toolchain = {'name': 'iccifort', 'version': '2019.5.281'}
+
+source_urls = ['https://www.lct.jussieu.fr/pagesperso/contrera/']
+sources = [{'filename': SOURCE_ZIP, 'download_filename': 'anci-RC.zip'}]
+checksums = ['d57c2969fc180c687c946a0a1ee393fe1ffc252b3534337bc71e4d95acbbf6dd']
+
+start_dir = '%(namelower)s'
+
+prebuildopts = "sed -i 's/include Makefile.inc//g' Makefile && "
+buildopts = 'LDFLAGS="$LDFLAGS $LIBS"'
```

This would place `$LIBS` before the declaration of objects which I think is wrong. The main error in the `Makefile` is that `$(LIBS)` is declared in the `nciplot` target for make to build `nciplot: $(OBJS) $(LIBS)`, which obviusly is going to fail as the libraries should not be build. Moreover, the definitions of `$LIBS`, `$INCLUDE` and `$LDFLAGS` in the `Makefile` step over any user defined values.

So, I suggest the following changes to the `Makefile`
```suggestion
prebuildopts += "sed -i 's/LDFLAGS=/override LDFLAGS+=/g' Makefile && "
prebuildopts += "sed -i 's/INCLUDE=/override INCLUDE+=/g' Makefile && "
prebuildopts += "sed -i 's/LIBS=/override LIBS+=/g' Makefile && "                    
prebuildopts += "sed -i 's/nciplot: $(OBJS) $(LIBS)/nciplot: $(OBJS)/g' Makefile && "
```
It is probably better to make a patch for the `Makefile` due to the number of changes.

---

you're right, thanks.
I've solved it slightly differently.

---

that works :+1: